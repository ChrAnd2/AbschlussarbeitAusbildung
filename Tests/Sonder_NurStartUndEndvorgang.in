//******************************************
//+ Sonderfall - Nur ein Start- und Endvorgang
//******************************************
//Vorgangsnummer; Vorgangsbezeichnung; Dauer; Vorgänger; Nachfolger
1; Problemanalyse; 2; -; 2
2; Grobplanung; 3; 1; -
