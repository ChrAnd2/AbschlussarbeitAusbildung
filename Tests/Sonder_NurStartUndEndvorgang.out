Sonderfall - Nur ein Start- und Endvorgang

Vorgangsnummer; Vorgangsbezeichnung; D; FAZ; FEZ; SAZ; SEZ; GP; FP
1; Problemanalyse; 2.0; 0.0; 2.0; 0.0; 2.0; 0.0; 0.0
2; Grobplanung; 3.0; 2.0; 5.0; 2.0; 5.0; 0.0; 0.0

Anfangsvorgang: 1
Endvorgang: 2
Gesamtdauer: 5.0

Kritischer Pfad:
1. 1->2
